website/identifiers/tasks.py

```diff
@@ -27,7 +27,14 @@ def task__update_verified_links(self, target_guid):
                  f'[guid={target_guid}]')
 
     Guid = apps.get_model('osf.Guid')
-    target_object = Guid.load(target_guid).referent
+    guid = Guid.load(target_guid)
+
+    if guid is not None:
+        target_object = guid.referent
+    else:
+        logger.error(f"Failed to load guid= {target_guid}")
```

Comment:
Let's do a sentry message log so we still catch the issue in sentry. If it turns out to happen more often than we expected, we may need to take a look at the GV side. For now, I think log to sentry is good enough.